### PR TITLE
build(relup): add new script for download relup base packages

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

**Review by terry-xiaoyu — APPROVED** [review]

**Review by terry-xiaoyu — APPROVED** [review]

File: scripts/relup-base-packages.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+SHASUM="sha256sum"
+if [ "$SYSTEM" = "macos" ]; then
+    SHASUM="shasum -a 256"
+fi
+
+mkdir -p _upgrade_base
+pushd _upgrade_base
+
+for tag in $(../scripts/relup-base-vsns.sh community | xargs echo -n); do
```
Comment (zmstone) [inline scripts/relup-base-packages.sh:52]:
```suggestion
for tag in $(../scripts/relup-base-vsns.sh $PROFILE | xargs echo -n); do
```

File: scripts/relup-base-packages.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    package_name="${PROFILE}-${tag#[e|v]}-otp${OTP_VSN}-${SYSTEM}-${ARCH}"
+
+    if [ ! -f "$package_name.tar.gz" ] \
+       && curl -I -m 10 -o /dev/null -s -w "%{http_code}" https://www.emqx.com/downloads/$DIR/"$tag"/"$package_name".tar.gz | grep -q -oE "^[23]+"
+    then
+        wget --no-verbose https://www.emqx.com/downloads/$DIR/"$tag"/"$package_name".tar.gz
+        if [ "$SYSTEM" != "centos6" ]; then
```
Comment (zmstone) [inline scripts/relup-base-packages.sh:59]: we do not have centos6 for 5.0

Comment (Rory-Z) [reply scripts/relup-base-packages.sh]: Cool !

File: scripts/relup-base-packages.sh (hunk 1 above)
Comment (Rory-Z) [reply scripts/relup-base-packages.sh]: Updated, we have updated the `relup-base-vsns.sh` script ?

Comment (zmstone) [reply scripts/relup-base-packages.sh]: no relup-base-vsns.sh is not changed,
5.0 enterprise edition has profile named `emqx-enterprise` matches the `*enterprise*` pattern.

**Review by zmstone — APPROVED** [review]

**Pull request merged**